### PR TITLE
Docs  advanced feb22

### PR DESCRIPTION
**Pull request by rivkap**
PR for updated advanced docs
including some updates from [PR 387](https://github.com/IBM/ibm-block-csi-driver/pull/387)

---
This change is [Reviewable](https://reviewable.io/reviews/ibm/ibm-block-csi-driver/453)

### CODE REVIEW

File: docs/advanced/advanced_troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  Verify that the CSI driver is running. (Make sure that all components are in the  _Running_ state), using the `$> kubectl get all -n <namespace> -l csi` command.
+
+## Multipath troubleshooting
+
+Use this information to help pinpoint potential causes for multipath failures.
+
+**Note:** Run these commands should be run on one of the worker nodes.
+
+-   **Display multipath information (FC and iSCSI)**
```
Comment (oriyarde) [inline docs/advanced/advanced_troubleshooting.md:18]: continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r805321064):
where does it say that "nvme is a part of FC configurations"?
I don't think we need to specify the types of connectivity for procedures that are connectivity agnostic anyway.
FYI @roysahar

File: docs/advanced/advanced_troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    Display multipath information, using the `sudo multipath -ll` command.
+
+    <pre>
+    mpathb (3600507680283851530000000000000a6) dm-0 IBM,2145
+    size=1.0G features='1 queue_if_no_path' hwhandler='1 alua' wp=rw
+    |-+- policy='service-time 0' prio=50 status=active
+    | `- 3:0:0:0 sda 8:0  active ready running
+    `-+- policy='service-time 0' prio=10 status=enabled
+    `- 2:0:0:0 sdb 8:16 active ready running
+
```
Comment (oriyarde) [inline docs/advanced/advanced_troubleshooting.md:29]: continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r805321106):
>i think i fixed this now

you can verify changes using `View file`:
image

File: docs/advanced/advanced_compatibility_reqs.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Compatibility and Requirements
+
+For iSCSI single path users (RHEL only) verify that multipathing is installed and running.
```
Comment (oriyarde) [inline docs/advanced/advanced_compatibility_reqs.md:3]: continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r805321132):
@roysahar could we let the storage documentation cover the multipath conf part, as we did until now?

File: docs/advanced/advanced_running_stateful.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+           
+            $> kubectl get statefulset/statefulset-name
+            Error from server (NotFound): statefulsets.apps <statefulset-name> not found
+
+    3. **For file system configuration only:** Verify that the multipath was deleted and that the volume device no longer exists in the output. Do this by establishing an SSH connection and logging into the worker node and using the following command sequence:
+          
+        <pre>
+        $> ssh root@k8s-node1
+            
+        $>kubectl get pvc -o jsonpath='{.items[*].metadata.annotations.volume\.beta\.kubernetes\.io\/storage-provisioner}' block.csi.ibm.com
```
Comment (oriyarde) [inline docs/advanced/advanced_running_stateful.md:44]: continuing [conversation](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r801367409):
@roysahar the k8s command cannot be run from a non-master node by default. please provide the relevant `df`/other command, or confirm the removal of this step

**Comment by oriyarde** [comment]
@rivkap this PR has been inactive for months, please change it to "draft", until it becomes active again

**Comment by kasserater** [comment]
very old PR (2 years old), assuming all the relevant documentation was changed in some way or another since

**Pull request closed without merging**